app/assets/javascripts/discourse/app/widgets/topic-admin-menu.js

```diff
@@ -343,13 +356,29 @@ export default createWidget("topic-admin-menu", {
       this.attrs,
       this.state
     );
-    return h(
-      "ul",
-      attrs.actionButtons
-        .concat(extraButtons)
-        .filter(Boolean)
-        .map((b) => this.attach("admin-menu-button", b))
+
+    const actionButtons = attrs.actionButtons.concat(extraButtons);
+
+    const buttonMap = actionButtons.reduce(
+      (prev, current) =>
+        prev.set(current.button_group, [
+          ...(prev.get(current.button_group) || []),
+          current,
+        ]),
+      new Map()
```

Comment:
This fails if there are any buttons added by plugins that don't have `button_group` attribute

Reply:
Scratch that. The issue is that `actionButtons` somehow can have `undefined` values:

<img width="275" alt="image" src="https://user-images.githubusercontent.com/66961/163673133-a640ab20-3bb3-4bce-883a-2041ea75950e.png">

So the error is:

```
TypeError: Cannot read properties of undefined (reading 'button_group')
    at topic-admin-menu.js:341:1
```